modules/sdk/project-templates/project-templates-cdi-bean-portlet/src/main/resources/archetype-resources/build.gradle

```diff
@@ -0,0 +1,43 @@
+#if (${projectType} != "workspace")buildscript {
+	dependencies {
+		classpath group: "com.liferay", name: "com.liferay.gradle.plugins", version: "@com.liferay.gradle.plugins.version@"
+	}
+
+	repositories {
+		maven {
+			url "https://repository-cdn.liferay.com/nexus/content/groups/public"
+		}
+	}
+}
+
+apply plugin: "com.liferay.plugin"
```

Comment:
Hey @ngriffin7a  take a look at the other war projects and see how their build.gradle are configured and see that they must apply the war plugin on the gradle side.  

also, have you been able to test this project template from both gradle and maven and compared the WAR files to make sure they are virtually identical?